index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport"
+    content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="welcome">Welcome</div>
+      <div class="menu-button">
+        <img src="resources/menu-icon.svg" alt="menu">
+      </div>
+    </nav>
+  </header>
+  <main class="headline-section">
+
+    <h1 class="hello">Hey there. I'm Soufiane
+      <br>
+      <span>
+        I'm a software developer
+      </span>
+    </h1>
+    <p class="supporting-text">I can help you build a
```

Comment:
[OPTIONAL]
Following HTML & CSS best practices, it would be nice to use the `section` tag for the headline section. The portfolio will have many sections in the future.

Reply:
Hi Amine!
Thank you for the feedback.
I did the changes that you asked for :)